From 31809f8419514f2130e9661d9989662165366e83 Mon Sep 17 00:00:00 2001
From: Stefan Dirsch <sndirsch@suse.de>
Date: Mon, 14 Mar 2022 11:38:23 +0100
Subject: [PATCH] meson: restore private requires to libdrm in dri.pc file

Due to a typo the private requires to libdrm were lost in dri.pc.
Fixed another typo (only comment).
---
 src/gallium/frontends/dri/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gallium/frontends/dri/meson.build b/src/gallium/frontends/dri/meson.build
index a0ae961c5fc..139707ea719 100644
--- a/src/gallium/frontends/dri/meson.build
+++ b/src/gallium/frontends/dri/meson.build
@@ -68,13 +68,13 @@ libdri = static_library(
 if with_dri
   dri_req_private = []
   if dep_libdrm.found()
-    dir_req_private = ['libdrm >= @0@'.format(_drm_ver)]
+    dri_req_private = ['libdrm >= @0@'.format(_drm_ver)]
   endif
 
   pkg.generate(
     name : 'dri',
     filebase : 'dri',
-    description : 'Direct Rendering Infastructure',
+    description : 'Direct Rendering Infrastructure',
     version : meson.project_version(),
     variables : ['dridriverdir=@0@'.format(dri_drivers_path)],
     requires_private : dri_req_private,
-- 
2.34.1

